### DISCUSSION TITLE
Adding a prefix even if no prefix is actually used on the code

### DISCUSSION THREAD

**Discussion by NeurAlch** [tailwindlabs/tailwindcss#6555] [Help]
Hey! I have React code that uses a function to add prefixes to the final output: `className={classes('container mx-auto')}`. 

This was to avoid having to re-write all classes to include the prefix, plus it is a pain to write them. The use case is for a Chrome Extension, so I need the prefixes to avoid collision on any site.

This is not a problem with Tailwind v2 since I can just set it to add a prefix. 

But tailwind 3 never finds those prefixes/classes, since it is looking for something like `__tw_container`, giving me an empty CSS file.

Is there a way to either:

* Get tailwind to output all classes (like in tailwind 2)
* Create the CSS classes with the prefixes even if the prefix is not actually used?

**Comment by camsloanftc** [comment]
I am in a similar situation. I would like to use tailwind for styling a widget, and thus I don't want to clash with any existing styles in the consuming app. It would be awesome to add a prefix to the generated styles without the need to use the prefixes throughout the codebase.
